src/routes/write-exercise/components/__tests__/__snapshots__/ArticleMissingPopoverContent.spec.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
```

Comment:
You can just remove it, we are planning to remove all snapshot tests.

Reply:
Will just keep it, not in scope of PR.